.pre-commit-config.yaml

```diff
@@ -0,0 +1,11 @@
+repos:
+  -   repo: https://github.com/ambv/black
+      rev: stable
+      hooks:
+        - id: black
+          language_version: python3.7
+  -   repo: https://gitlab.com/pycqa/flake8
+      rev: 3.7.9
+      hooks:
+        - id: flake8
+          args: ['--max-line-length=88']
```

Comment:
Since it defined in the root, flake is not aware of /{presidio-service}/setup.cfg flake8 configuration.
This mean we either have to repeat the config here.
I've tried to define a global flake8 configuration (makes sense since we have duplication between services anyway) in the project root but it was ignored from some reason. @omri374 do you know if its doable?